### DISCUSSION TITLE
Web cam putting  over 20 foot putts

### DISCUSSION THREAD

**Discussion by gerardkele** [springbok/MLM2PRO-GSPro-Connector#96] [Q&A]
GS pro Putting is working great until I get to long putts or very up hill putts. Anything which needs a good bit of speed right off the putter face. I have it set up to read 11 footer spot on, fall in cup and on screen at the same pace. Once it get out to 25 feet nothing really rolls much more than 4 foot on screen. Or when I try the putt putt courses with kids. Any thoughts.

**Comment by springbok** [comment]
Sorry I don't know the answer, maybe ask Alex at https://github.com/alleexx](https://github.com/alleexx/cam-putting-py

**Reply by gerardkele** [reply]
Will do thanks
